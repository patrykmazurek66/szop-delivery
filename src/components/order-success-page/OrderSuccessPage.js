import { FavoriteOutlined } from "@mui/icons-material";
import { Button } from "@mui/material";
import { Link } from "react-router-dom";
import Footer from "../footer/Footer";
import "./ordersuccesspage.css";

export default function OrderSuccessPage() {
  return (
    <div className="order-success-page">
      <div className="order-success-page--hero">
        <div className="order-success-page--hero--info-container">
          <p>Ordered successfully!</p>
          <FavoriteOutlined fontSize="50px" />
          <Link to="/">
            <Button variant="contained">Back to Main Page</Button>
          </Link>
        </div>
      </div>
      <Footer />
    </div>
  );
}
