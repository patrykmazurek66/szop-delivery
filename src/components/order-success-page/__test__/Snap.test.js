import { BrowserRouter } from "react-router-dom";
import renderer from "react-test-renderer";
import OrderSuccessPage from "../OrderSuccessPage";

it("renders correctly", () => {
  const tree = renderer
    .create(
      <BrowserRouter>
        <OrderSuccessPage />
      </BrowserRouter>
    )
    .toJSON();
  expect(tree).toMatchSnapshot();
});
