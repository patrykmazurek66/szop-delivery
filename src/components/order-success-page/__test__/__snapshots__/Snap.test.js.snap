// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div
  className="order-success-page"
>
  <div
    className="order-success-page--hero"
  >
    <div
      className="order-success-page--hero--info-container"
    >
      <p>
        Ordered successfully!
      </p>
      <svg
        aria-hidden={true}
        className="MuiSvgIcon-root MuiSvgIcon-fontSize50px css-821wyw-MuiSvgIcon-root"
        data-testid="FavoriteOutlinedIcon"
        focusable="false"
        viewBox="0 0 24 24"
      >
        <path
          d="m12 21.35-1.45-1.32C5.4 15.36 2 12.28 2 8.5 2 5.42 4.42 3 7.5 3c1.74 0 3.41.81 4.5 2.09C13.09 3.81 14.76 3 16.5 3 19.58 3 22 5.42 22 8.5c0 3.78-3.4 6.86-8.55 11.54L12 21.35z"
        />
      </svg>
      <a
        href="/"
        onClick={[Function]}
      >
        <button
          className="MuiButtonBase-root MuiButton-root MuiButton-contained MuiButton-containedPrimary MuiButton-sizeMedium MuiButton-containedSizeMedium MuiButton-root MuiButton-contained MuiButton-containedPrimary MuiButton-sizeMedium MuiButton-containedSizeMedium css-sghohy-MuiButtonBase-root-MuiButton-root"
          disabled={false}
          onBlur={[Function]}
          onContextMenu={[Function]}
          onDragLeave={[Function]}
          onFocus={[Function]}
          onKeyDown={[Function]}
          onKeyUp={[Function]}
          onMouseDown={[Function]}
          onMouseLeave={[Function]}
          onMouseUp={[Function]}
          onTouchEnd={[Function]}
          onTouchMove={[Function]}
          onTouchStart={[Function]}
          tabIndex={0}
          type="button"
        >
          Back to Main Page
        </button>
      </a>
    </div>
  </div>
  <footer>
    <div
      className="footer-gap"
    />
    <div
      className="footer-content"
    >
      <div
        className="footer--links-container"
      >
        <a
          href="/#"
        >
          Help
        </a>
        <a
          href="/#"
        >
          FAQ
        </a>
        <a
          href="/#"
        >
          Cookies
        </a>
        <a
          href="/#"
        >
          Contact
        </a>
        <a
          href="/#"
        >
          Facebook
        </a>
        <a
          href="/#"
        >
          Log in
        </a>
        <a
          href="/#"
        >
          Sign up
        </a>
      </div>
      <p>
        Â© SzopDelivery 2022
      </p>
    </div>
  </footer>
</div>
`;
